tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -54,12 +59,29 @@ describe(__filename, () => {
     expect(root).toHaveClassName('ShowMoreCard--expanded');
   });
 
-  it('is expanded by default', () => {
+  it('is expanded by default if content is not too long', () => {
     const root = render();
 
     expect(root).toHaveClassName('ShowMoreCard--expanded');
   });
 
+  it('is truncate by default if content is too long', () => {
+    const { store } = dispatchClientMetadata();
+
+    const root = render({
+      store,
+      shallowOptions: { disableLifecycleMethods: true },
+    });
+
+    root.instance().contents = { clientHeight: MAX_HEIGHT + 1 };
+
+    root.instance().componentDidMount();
```

Comment:
This test is pretty hacky, but I cannot think of a better way to do it myself, so I guess it's ok.